test/parallel/test-runner-mock-timers.js

```diff
@@ -111,14 +112,34 @@ describe('Mock Timers Test Suite', () => {
         assert.strictEqual(timeoutFn.mock.callCount(), 1);
         assert.strictEqual(intervalFn.mock.callCount(), 1);
       });
-    });
 
+      it('should increase the epoch as the tick run for runAll', async (t) => {
+        const timeoutFn = t.mock.fn();
+        const intervalFn = t.mock.fn();
+
+        t.mock.timers.enable();
+        global.setTimeout(timeoutFn, 1111);
+        const id = global.setInterval(intervalFn, 9999);
+        t.mock.timers.runAll();
+
+        global.clearInterval(id);
+        assert.strictEqual(timeoutFn.mock.callCount(), 1);
+        assert.strictEqual(intervalFn.mock.callCount(), 1);
+        assert.strictEqual(Date.now(), 9999);
+      });
+
+      it('should not error if there are not timers to run', (t) => {
+        t.mock.timers.enable();
+        t.mock.timers.runAll();
+        // Should not throw
```

Comment:
use assert.doestNotThrow https://nodejs.org/api/assert.html#assertdoesnotthrowfn-error-message

Reply:
There's an [ESLint](https://github.com/nodejs/node/blob/77ea1732a1da23719c3adc016629ef3a179803c8/test/.eslintrc.yaml#L23) rule that prevents me from using it. It says I should just write the code and put a comment in it.